local-tests/tests/testEthereumBroadcastTransactionGeneratedKey.ts

```diff
@@ -31,7 +28,7 @@ export const testEthereumBroadcastTransactionGeneratedKey = async (
 
   const { pkpAddress, generatedPublicKey } = await generatePrivateKey({
     pkpSessionSigs,
-    network: NETWORK_EVM,
+    network: 'evm',
```

Comment:
So once we started restricting `network` to be specific values using a string union type, I discovered that if someone used the `const` to call our method, the typescript checker would throw an error `TS2322: Type string is not assignable to type Network` -- because the type of the constant is `string`, decides that it is too generate -- it's not a _specific string literal_.

So then I tried defining these as:
```javascript
export const NETWORK_EVM: Network = 'evm';
export const NETWORK_SOLANA: Network = 'solana';
```
That _did_ fix the consumers code's typechecking, but then, _inside the SDK_ I started getting TS checker errors in the `lit-actions-client/constants.ts` file where we define the `LIT_ACTION_CID_REPOSITORY`:

`TS2739: Type
Readonly<{   [x: string]: 'QmdYUhPCCK5hpDWMK1NiDLNLG6RZQy61QE4J7dBm1Y2nbA' | 'QmSi9GL2weCFEP1SMAUw5PDpZRr436Zt3tLUNrSECPA5dT'; }>
is missing the following properties from type Readonly<Record<Network, string>>: evm, solana`

Even though the entry clearly has the correct keys in it, defined by their constants :(
```javascript
import { LitCidRepository } from './types';
import { NETWORK_EVM, NETWORK_SOLANA } from '../constants';

import { Network } from '../types';

export type LitActionType =
  | 'signTransaction'
  | 'signMessage'
  | 'generateEncryptedKey';

export type LitCidRepositoryEntry = Readonly<Record<Network, string>>;

export type LitCidRepository = Readonly<
  Record<LitActionType, LitCidRepositoryEntry>
>;

const LIT_ACTION_CID_REPOSITORY: LitCidRepository = {
  signTransaction: Object.freeze({
    [NETWORK_EVM]: 'QmdYUhPCCK5hpDWMK1NiDLNLG6RZQy61QE4J7dBm1Y2nbA',
    [NETWORK_SOLANA]: 'QmSi9GL2weCFEP1SMAUw5PDpZRr436Zt3tLUNrSECPA5dT',
  }),
  signMessage: Object.freeze({
    [NETWORK_EVM]: 'QmTMGcyp77NeppGaqF2DmE1F8GXTSxQYzXCrbE7hNudUWx',
    [NETWORK_SOLANA]: 'QmYQC6cd4EMvyB4XPkfEEAwNXJupRZWU5JsTCUrjey4ovp',
  }),
  generateEncryptedKey: Object.freeze({
    [NETWORK_EVM]: 'QmaoPMSqcze3NW3KSA75ecWSkcmWT1J7kVr8LyJPCKRvHd',
    [NETWORK_SOLANA]: 'QmdRBXYLYvcNHrChmsZ2jFDY8dA99CcSdqHo3p1ES3UThL',
  }),
};

export { LIT_ACTION_CID_REPOSITORY };
```

In the end, I discovered that I had to use _string literals_ in the LIT_ACTION_CID_REPOSITORY to make TS happy; I couldn't use our consts as keys.  Then I was faced with a conundrum; now external consumers would be using constants that we didn't use ourselves.  Since TS can't use runtime data as types, I went the opposite direction and exported the string union type. I did some digging around the net to try and identify a solution that would let me use the same consts that consumers used to define the keys on our repository, but couldn't find a solution and it was taking quite a lot of time at that point.

Given that these are user-facing arguments, maybe it'd be fine for our internal files to use string constants when defining actual records, but external consumers still have constants to use if they want to?  I've pushed a commit that makes that change -- take a look and LMK  what you think.  I also added exported constants for `keyType` using the same pattern.